internal/renderer/render.go

```
@@ -0,0 +1,259 @@
+			"-flags:a", "+bitexact",
+		)
+	}
+	if req.Output.Codec != "h264" {
+		return fmt.Errorf("renderer: codec %q 不受支持（Phase 0 仅 h264）", req.Output.Codec)
+	}
+	args = append(args, "-movflags", "+faststart", req.Output.Path)
+
+	if err := os.MkdirAll(filepath.Dir(req.Output.Path), 0o755); err != nil {
+		return fmt.Errorf("renderer: 建输出目录: %w", err)
+	}
+	cmd := exec.Command(ffmpeg, args...)
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)